packages/apputils-extension/src/announcements.ts

```diff
@@ -266,7 +266,9 @@ export const announcements: JupyterFrontEndPlugin<void> = {
                   callback: () => {
                     window.open(link[1], '_blank', 'noreferrer');
                   },
-                  displayType: 'link'
+                  // Because the link to the changelog is the primary option,
+                  // display it in an accent color.
+                  displayType: 'accent'
```

Comment:
FYI: The news pop-up mention there is no primary color used. One reason I did not use it is to reduce visual noise for the pop-up.